Makefile

```diff
@@ -138,39 +162,57 @@ agent: cmd/agent/agent
 agentctl: cmd/agentctl/agentctl
 agent-operator: cmd/agent-operator/agent-operator
 
-cmd/agent/agent: check-seego cmd/agent/main.go
+
+
+
+# In general DRONE variable should overwrite any other options, if DRONE is not set then fallback to normal behavior
+
+cmd/agent/agent: check-seego  cmd/agent/main.go
+ifeq ($(DRONE),true)
+	$(CGO_CC_CCX); go build $(CGO_FLAGS) -o $@ ./$(@D)
+else
 ifeq ($(CROSS_BUILD),false)
 	CGO_ENABLED=1 go build $(CGO_FLAGS) -o $@ ./$(@D)
 else
-	@CGO_ENABLED=1 GOOS=$(GOOS) GOARCH=$(GOARCH) GOARM=$(GOARM); $(seego) build $(CGO_FLAGS) -o $@ ./$(@D)
+	CGO_ENABLED=1 GOOS=$(GOOS) GOARCH=$(GOARCH) GOARM=$(GOARM); $(seego) build $(CGO_FLAGS) -o $@ ./$(@D)
+endif
 endif
 	$(NETGO_CHECK)
 
+
 cmd/agentctl/agentctl: check-seego cmd/agentctl/main.go
+ifeq ($(DRONE),true)
+	$(CGO_CC_CCX); go build $(CGO_FLAGS) -o $@ ./$(@D)
+else
 ifeq ($(CROSS_BUILD),false)
 	CGO_ENABLED=1 go build $(CGO_FLAGS) -o $@ ./$(@D)
 else
-	@CGO_ENABLED=1 GOOS=$(GOOS) GOARCH=$(GOARCH) GOARM=$(GOARM); $(seego) build $(CGO_FLAGS) -o $@ ./$(@D)
+	CGO_ENABLED=1 GOOS=$(GOOS) GOARCH=$(GOARCH) GOARM=$(GOARM); $(seego) build $(CGO_FLAGS) -o $@ ./$(@D)
```

Comment:
Can we leave this as-is?

Reply:
I personally like seeing all the output on builds, makes it easier to debug. For someone who hasn't worked in make in a while or hasn't, it is hard to know what is wrong when you don't realize the output isn't being written out. (Like I was initially). It does add considerable noise though.